spec/system/task_spec.rb

```diff
@@ -1,24 +1,25 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
```

Comment:
設定の追記で省略できるので、全体のコード量を減らして見通しをよくするために積極的に省略しましょう。
```suggestion
  let(:project) { create(:project) }
```